### PR TITLE
feat: trajectory follower package

### PR DESCRIPTION
**Pull request by taikitanaka3**
Related Issue
#1

Related Link
This PR depends on #4

Description
Add trajectory follower nodes packages from https://github.com/tier4/AutowareArchitectureProposal.iv

this PR depends
https://github.com/autowarefoundation/autoware.universe/pull/83

How to review
Check the following items

- readme
- sample launch

### CODE REVIEW

File: control/trajectory_follower_nodes/src/lateral_controller_node.cpp [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+    parameters.cbegin(), parameters.cend(),
+    [&name](const rclcpp::Parameter & parameter) { return parameter.get_name() == name; });
+  if (it != parameters.cend()) {
+    value = static_cast<T>(it->template get_value<T>());
+  }
+}
+}  // namespace
+
+LateralController::LateralController(const rclcpp::NodeOptions & node_options)
+: Node("lateral_controller", node_options)
```
Comment (1222-takeshi) [inline control/trajectory_follower_nodes/src/lateral_controller_node.cpp:53]: Could you create sample launch file?

File: control/trajectory_follower_nodes/src/latlon_muxer_node.cpp [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+namespace autoware
+{
+namespace motion
+{
+namespace control
+{
+namespace trajectory_follower_nodes
+{
+LatLonMuxer::LatLonMuxer(const rclcpp::NodeOptions & node_options)
+: rclcpp::Node("latlon_muxer", node_options)
```
Comment (1222-takeshi) [inline control/trajectory_follower_nodes/src/latlon_muxer_node.cpp:29]: Could you create sample launch file?

File: control/trajectory_follower_nodes/src/longitudinal_controller_node.cpp [hunk 3]
```diff
@@ -0,0 +1,936 @@
...
+namespace autoware
+{
+namespace motion
+{
+namespace control
+{
+namespace trajectory_follower_nodes
+{
+LongitudinalController::LongitudinalController(const rclcpp::NodeOptions & node_options)
+: Node("longitudinal_controller", node_options)
```
Comment (1222-takeshi) [inline control/trajectory_follower_nodes/src/longitudinal_controller_node.cpp:37]: Could you create sample launch file?

**Comment by tkimura4** [comment]
Could you revert pre-commit and append this package to ignore list of pre-commit?

File: control/trajectory_follower_nodes/src/lateral_controller_node.cpp (hunk 1 above)
Comment (taikitanaka3) [reply control/trajectory_follower_nodes/src/lateral_controller_node.cpp:53]: these are a packages porting from .auto so changes applied to them should be as minimal as possible.
https://github.com/autowarefoundation/autoware.universe/issues/117

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**